src/sentry/grouping/ingest/seer.py

```diff
@@ -212,13 +216,34 @@ def get_seer_similar_issues(
     event_hash = event.get_primary_hash()
     exception_type = get_path(event.data, "exception", "values", -1, "type")
 
+    stacktrace_string = event.data.get(
+        "stacktrace_string",
+        get_stacktrace_string_with_metrics(
+            get_grouping_info_from_variants(variants), event.platform, ReferrerOptions.INGEST
+        ),
+    )
+
+    if not stacktrace_string:
+        # TODO: remove this log once we've confirmed it isn't happening
+        logger.info(
+            "get_seer_similar_issues.empty_stacktrace",
+            extra={
+                "event_id": event.event_id,
+                "project_id": event.project.id,
+                "stacktrace_string": stacktrace_string,
+            },
+        )
+        similar_issues_metadata_empty = {
+            "results": [],
+            "similarity_model_version": SEER_SIMILARITY_MODEL_VERSION,
+        }
+        return (similar_issues_metadata_empty, None)
```

Comment:
Added another null check for `stacktrace_string` here. This shouldn't happen anymore due to the fix mentioned in the previous comment, but I included it as a fallback